config/locales/en.yml

```diff
@@ -28,10 +28,10 @@ en:
   edit_user: "Edit User"
   delete: "Delete"
   fix_errors_before_continuing: "You need to fix the errors on this page before continuing."
-  listing_organisations: "Listing Organisations"
-  listing_profiles: "Listing Profiles"
-  listing_roles: "Listing Roles"
-  listing_users: "Listing Users"
+  organisations: "Organisations"
+  listing_profiles: "Profiles"
```

Comment:
change the names from `listing_profiles` to `profiles` etc if you are changing the text :)